src/azure-cli/azure/cli/command_modules/network/_template_builder.py

```diff
@@ -5,7 +5,7 @@
 
 
 def _build_frontend_ip_config(cmd, name, public_ip_id=None, subnet_id=None, private_ip_address=None,
-                              private_ip_allocation=None, zone=None):
+                              private_ip_allocation=None, zone=None, private_ip_address_version=None):
```

Comment:
Why not declare `--private_ip_address_version` for `az network lb frontend-ip` command in _params.py?

Reply:
declared it for `az network lb`